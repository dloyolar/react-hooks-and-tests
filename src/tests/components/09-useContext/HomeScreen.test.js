import { mount } from 'enzyme';
import React from 'react';
import { HomeScreen } from '../../../components/09-useContext/HomeScreen';
import { UserContext } from '../../../components/09-useContext/UserContext';

describe('Test in <HomeScreen />', () => {
  const user = {
    name: 'Diego',
    email: 'dieegolr@gmail.com',
  };

  const wrapper = mount(
    <UserContext.Provider value={{ user }}>
      <HomeScreen />
    </UserContext.Provider>
  );

  test('should be display correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
