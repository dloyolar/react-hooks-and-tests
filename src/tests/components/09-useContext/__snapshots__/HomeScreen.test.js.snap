// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test in <HomeScreen /> should be display correctly 1`] = `
<div>
  <h1>
    Home Screen
  </h1>
  <hr />
  <pre>
    {
   "name": "Diego",
   "email": "dieegolr@gmail.com"
}
  </pre>
</div>
`;
